tchannel/tornado/connection.py

```diff
@@ -121,7 +120,13 @@ def __init__(self, connection, tchannel=None):
         self._loop_running = False
 
         self.tchannel = tchannel
+        self.incoming_requests = {}
+        self.incoming_responses = {}
```

Comment:
I don't understand what we would put in the 'outgoing_request' map and why that isn't covered by considering it in-flight.

What do these maps actually represent? It's not clear from the code. What's the key/value type? Are they fragmented messages we're building? Are they requests waiting for responses?